packages/devextreme/scss/widgets/generic/menu/_index.scss

```diff
@@ -34,7 +34,9 @@
 .dx-menu {
   color: $menu-color;
 
-  .dx-menu-item-expanded {
+  .dx-menu-item-expanded,
+  .dx-menu-item-expanded.dx-state-focused {
```

Comment:
seems like the solution works even without this selector (`.dx-menu-item-expanded` is enough)

Reply:
?

Reply:
If we don't see a difference in the behavior and idealogical advantages of this addition let's get rid of it.